api/services/user/src/actions/DeleteAssociatedUserAction.ts

```diff
@@ -0,0 +1,31 @@
+import { Action as AbstractAction } from '@ilos/core';
+import { handler } from '@ilos/common';
+
+import { handlerConfig, ParamsInterface, ResultInterface } from '../shared/user/deleteAssociated.contract';
+import { alias } from '../shared/user/deleteAssociated.schema';
+import { UserRepositoryProviderInterfaceResolver } from '../interfaces/UserRepositoryProviderInterface';
+
+/*
+ *  Find user by id and delete user
+ */
+@handler({
+  ...handlerConfig,
+  middlewares: [
+    ['validate', alias],
+    // ['channel.service.only', ['territory', 'operator']],
+  ],
```

Comment:
Il manque le middleware d'authorization (commenté)

```suggestion
    ['validate', alias],
    ['channel.service.only', ['territory', 'operator']],
  ],
```